WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -444,7 +467,7 @@ private async Task MonitorAndHandlingCoinJoinFinalizationAsync(ConcurrentDiction
 
 			CoinJoinClientStates = GetCoinJoinClientStates(wallets, trackedCoinJoins, trackedAutoStarts);
 			CoinsInCriticalPhase = GetCoinsInCriticalPhase(wallets, trackedCoinJoins);
-			RoundStatusUpdater.SlowRequestsMode = HighestCoinJoinClientState is CoinJoinClientState.Idle;
+			RoundStatusUpdater!.SlowRequestsMode = HighestCoinJoinClientState is CoinJoinClientState.Idle;
```

Comment:
I believe it is ok to use `!` because if it is null then it means there is an issue somewhere and we should throw.

Reply:
I don't think so. Please consider this: https://github.com/WalletWasabi/WalletWasabi/commit/dc71b6fd9736922fd5bc31123f57c6384fb17682